packages/console/src/monaco/MonacoUtils.ts

```diff
@@ -339,91 +340,60 @@ class MonacoUtils {
     return platform.startsWith('Mac');
   }
 
+  // Tracks whether removeConflictingKeybindings() has been called
+  static conflictingKeybindingsRemoved = false;
+
   /**
    * Remove any keybindings which are used for our own shortcuts.
-   * This allows the key events to bubble up so a component higher up can capture them
-   * @param editor The editor to remove the keybindings from
+   * This allows the key events to bubble up so a component higher up can capture
+   * them. Note that this is a global configuration, so all editor instances will
+   * be impacted.
    */
-  static removeConflictingKeybindings(
-    editor: monaco.editor.IStandaloneCodeEditor
-  ): void {
-    // Multi-mod key events have a specific order
-    // E.g. ctrl+alt+UpArrow is not found, but alt+ctrl+UpArrow is found
-    // meta is WindowsKey on Windows and cmd on Mac
-    // ctrl is ctrl Windows and ctrl on Mac
-    // alt is alt on Windows and option on Mac
-    const keybindings = [
+  static removeConflictingKeybindings(): void {
+    // This function adds the keybindings to the global registry, so we only
+    // want to do it once
+    if (MonacoUtils.conflictingKeybindingsRemoved) {
+      return;
+    }
+
+    MonacoUtils.conflictingKeybindingsRemoved = true;
+
+    /* eslint-disable no-bitwise */
+    monaco.editor.addKeybindingRules([
```

Comment:
I was kind of confused about difference between this and `disableKeybindings`. May want to add a note in the docs stating it applies globally for all monaco editor instances.